### ISSUE TITLE
PBT 2.1.2 error: PBT cannot find a Pressbooks install

### ISSUE THREAD

**Issue by JackDougherty** [BCcampus/pressbooks-textbook#50]
Brad,
I created a fresh install on a test site as described below, but received an error.
- New WP 4.1.1 site and database, set to multisite
- Installed PB 3.1.2 from GitHub release, and successfully created a test book
- Installed PB 2.1.2 from GitHub release
- Network Dashboard error: "PBT cannot find a Pressbooks install"
- Also, the Pressbooks Textbook theme does not appear in Network Dashboard > Themes
Let me know if you would like privileges to explore this public test site.

![dashboard_ _pressbookstest_ _pressbooks](https://cloud.githubusercontent.com/assets/649719/12379660/963a405a-bd2c-11e5-98d7-09b302fce258.png)

thanks,
Jack

**Comment by bdolor** [comment]
Hi Jack, 

That admin notice gets triggered if PBT can't find /pressbooks/compatibility.php — see here https://github.com/BCcampus/pressbooks-textbook/blob/dev/pressbooks-textbook.php#L407

**Comment by JackDougherty** [comment]
Hi Brad -- Thanks, as I didn't expect to receive a response on a Sunday afternoon, and my question certainly can wait.

I don't know why the compatibility.php notice is triggered. This server is running PHP 7.0.2, and PB runs fine. Anything else I should check?

Tried a second installation and received the same error. Here's the public testbook:
http://pressbookstest2.trinfocafe.org/testbook/

**Comment by JackDougherty** [comment]
Brad - here's more info, whenever you have time to look at it:

I created a new local install on my Mac laptop, running MAMP 3.5 with PHP 7.0.0, and the same WP/PB/PBT versions as above. 

This time I didn't see the "PBT cannot find a PB install" error. However, even though PBT is installed and network activated, the PBT theme does not appear anywhere in the network themes panel. This is the same behavior as above. See screenshot. 

![themes_ _network_admin__wppbmamp_ _pressbooks](https://cloud.githubusercontent.com/assets/649719/12380292/5ce1d050-bd3d-11e5-8543-625f90bc24f6.png)

**Comment by bdolor** [comment]
This morning I took the steps outlined above to try to recreate the problem. I installed a fresh version of PB 3.1.2, PBT 2.1.2 and WP 4.4.1 on my local machine. Unfortunately I am not able to produce the same results that are described above — no error message that it could not find a PB install and that the OTB theme was present after network activating the PBT plugin. 

There are differences between the server environments that we are both using and I notice a few code changes have been committed to PB 3.1.2 since you tried it yesterday. I'm wondering if these differences are worth investigating as possible causes for the behaviour you are seeing?

**Comment by JackDougherty** [comment]
Brad -- Thanks, as always, for your suggestions. But I'm still stumped. I tried a fresh install on my local MAMP using PB and PBT master zips from GitHub. Yet I'm seeing the exact same error as above, where PB runs fine, but does not play nicely with PBT, at least for me. I will keep trying other variations to try to figure out why it's not working for me.

**Comment by bdolor** [comment]
Thanks Jack, I wish I could be more helpful and get this resolved for you. I would look to server error messages if they're producing anything meaningful and file permissions first. I would try to eliminate the server/server config as a possible source of the problem. 

If anyone else is experiencing this same problem, I encourage you to post here.

**Comment by JackDougherty** [comment]
Ran another test. This older combination from Sept 2015 runs perfectly on a fresh install on my local MAMP 3.5 (with PHP 7.0.0):
- WP 4.3.1
- PB 2.7.2
- PBT 1.2.16
The Open Textbooks theme appears, with no errors.
Will keep trying . . .

**Comment by JackDougherty** [comment]
. . . and when I upgraded the older combo (WP 4.3.1/PB 2.7.2/PB 1.2.16) on my local MAMP to the current versions, everything works.

First I upgraded WP thru the dashboard, then PB thru the dashboard. PBT got stuck when I tried to upgrade it this way, so I manually installed it in the plugins folder. 

So I still don't know why my fresh install of the newer versions do not yet work for me, even though I manually installed those in the plugins folder. 

I'll understand if you decide to close out this issue. . . but will continue scratching my head in puzzlement, and will try to replicate this successful September-to-January upgrade on a live test server.
